Default.sublime-commands

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "caption": "SublimeLinter XO: Fix current file",
+    "command": "xofix"
```

Comment:
```suggestion
    "command": "xo-fix"
```

Reply:
I changed the `XofixCommand` to `XoFixCommand` but for some reason the command doesn't work anyways.
The command `xo_fix` worked, so we can use that one, it's ok?

By the way, I was just following the same convention as Go:
https://github.com/noonat/sublime-gofmt/blob/master/Default.sublime-commands#L4

Reply:
> The command xo_fix worked, so we can use that one, it's ok?

👍